SimmyDemo_WebApi/Chaos/SimmyExtensions.cs

```diff
@@ -34,19 +36,18 @@ public static IPolicyRegistry<string> AddChaosInjectors(this IPolicyRegistry<str
                 if (policyEntry.Value is IAsyncPolicy<HttpResponseMessage> policy)
                 {
                     registry[policyEntry.Key] = policy
-                            .WrapAsync(MonkeyPolicy.InjectFaultAsync<HttpResponseMessage>(
-                                GetException,
-                                GetInjectionRate,
-                                GetEnabled))
-                            .WrapAsync(MonkeyPolicy.InjectFaultAsync<HttpResponseMessage>(
-                                GetHttpResponseMessage,
-                                GetInjectionRate,
-                                GetHttpResponseEnabled))
-                            .WrapAsync(MonkeyPolicy.InjectLatencyAsync<HttpResponseMessage>(
-                                GetLatency,
-                                GetInjectionRate,
-                                GetEnabled))
-                        ;
+                            .WrapAsync(MonkeyPolicy.InjectExceptionAsync(with =>
+                                with.Fault<HttpResponseMessage>(GetException)
+                                    .InjectionRate(GetInjectionRate)
+                                    .EnabledWhen(GetEnabled)))
+                            .WrapAsync(MonkeyPolicy.InjectResultAsync<HttpResponseMessage>(with =>
+                                with.Result(GetHttpResponseMessage)
+                                    .InjectionRate(GetInjectionRate)
+                                    .EnabledWhen(GetEnabled)))
```

Comment:
This used `GetHttpResponseEnabled` in the deleted code. Should it still?

Reply:
OMG! Good Catch Dylan, it was a copy/paste mistake 😅